intel_extension_for_transformers/transformers/modeling/modeling_auto.py

```diff
@@ -54,38 +54,26 @@
     WEIGHTS_INDEX_NAME,
     SAFE_WEIGHTS_NAME,
 )
-from ...llm.quantization.utils import (
-    convert_dtype_str2torch,
-    convert_dtype_torch2str,
-    convert_to_quantized_model,
-    replace_linear
-)
+from ...llm.quantization.utils import (convert_dtype_str2torch, convert_dtype_torch2str, convert_to_quantized_model,

```

Comment:
Why format changes？

Reply:
Yes. I used the yapf format tool to format this file, but it might not be the same as the previous code style.

I have reverted this format commit.